src/applications/vaos/utils/error.js

```diff
@@ -1,13 +1,24 @@
 import * as Sentry from '@sentry/browser';
+import { recordVaosError } from './events';
+
+export function captureError(err, skipRecordEvent) {
+  let eventErrorKey;
 
-export function captureError(err) {
   if (err instanceof Error) {
     Sentry.captureException(err);
+    eventErrorKey = err.message;
   } else {
     Sentry.withScope(scope => {
       scope.setExtra('error', err);
+      const errorTitle = err?.[0]?.title;
+      const message = `vaos_server_error${errorTitle ? `: ${errorTitle}` : ''}`;
+      eventErrorKey = message;
```

Comment:
Made this a little clearer, but the reason it wasn't as concise before is because I didn't want to include a colon after `vaos_server_error` unless we had an error title.

Reply:
I think it needs to be `err?.errors?.[0]?.title`. Though I may end up changing this to break out the errors by url, later, to make Sentry clearer.